### ISSUE TITLE
NoClassDefFoundError org.apache.commons.compress.utils.Lists

### ISSUE THREAD

**Issue by DAQEM** [jayhillx/MysticsBiomes#22]
I'm getting the following error when running the mod on a Forge Server.

```console
java.lang.NoClassDefFoundErrororg/apache/commons/compress/utils/Lists
[03.03 15:36:13] [Server] 	at com.mysticsbiomes.common.animal.RainbowChicken.<init>(RainbowChicken.java:50) ~[mysticsbiomes-1.20.1-3.1.6.jar%23278!/:3.1.{re:classloading}
[03.03 15:36:13] [Server] 	at net.minecraft.world.entity.EntityType.m_20615_(EntityType.java:525) ~[server-1.20.1-20230612.114412-srg.jar%23319!/:?] {re:mixin,xf:fml:forge:forge_method_redirector,re:classloading,xf:fml:forge:forge_method_redirector,pl:mixin:APP:architectury-common.mixins.json:inject.MixinEntityType,pl:mixin:A}
[03.03 15:36:13] [Server] 	at net.minecraft.world.level.NaturalSpawner.m_220450_(NaturalSpawner.java:363) ~[server-1.20.1-20230612.114412-srg.jar%23319!/:?] {re:mixin,xf:fml:forge:forge_method_redirector,re:classloading,xf:fml:forge:forge_method_redirector,pl:mixin:APP:botania_xplat.mixins.json:NaturalSpawnerMixin,pl:mixin:A}
[03.03 15:36:13] [Server] 	at net.minecraft.world.level.levelgen.NoiseBasedChunkGenerator.m_6929_(NoiseBasedChunkGenerator.java:453) ~[server-1.20.1-20230612.114412-srg.jar%23319!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:citadel.mixins.json:NoiseBasedChunkGeneratorMixin,pl:mixin:A}
[03.03 15:36:13] [Server] 	at net.minecraft.world.level.chunk.ChunkStatus.m_196757_(ChunkStatus.java:140) ~[server-1.20.1-20230612.114412-srg.jar%23319!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:citadel.mixins.json:ChunkStatusMixin,pl:mixin:A}
[03.03 15:36:13] [Server] 	at net.minecraft.world.level.chunk.ChunkStatus$SimpleGenerationTask.m_214024_(ChunkStatus.java:309) ~[server-1.20.1-20230612.114412-srg.jar%23319!/:?] {re:classloading}
[03.03 15:36:13] [Server] 	at net.minecraft.world.level.chunk.ChunkStatus.m_280308_(ChunkStatus.java:252) ~[server-1.20.1-20230612.114412-srg.jar%23319!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:citadel.mixins.json:ChunkStatusMixin,pl:mixin:A}
[03.03 15:36:13] [Server] 	at net.minecraft.server.level.ChunkMap.lambda$scheduleChunkGeneration$27(ChunkMap.java:643) ~[server-1.20.1-20230612.114412-srg.jar%23319!/:?] {re:classloading,pl:accesstransformer:B}
[03.03 15:36:13] [Server] 	at com.mojang.datafixers.util.Either$Left.map(Either.java:38) ~[datafixerupper-6.0.8.jar%2377!/:?] {}
[03.03 15:36:13] [Server] 	at net.minecraft.server.level.ChunkMap.lambda$scheduleChunkGeneration$29(ChunkMap.java:634) ~[server-1.20.1-20230612.114412-srg.jar%23319!/:?] {re:classloading,pl:accesstransformer:B}
[03.03 15:36:13] [Server] 	at java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1150) ~[?:?] {}
[03.03 15:36:13] [Server] 	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?] {}
[03.03 15:36:13] [Server] 	at net.minecraft.server.level.ChunkTaskPriorityQueueSorter.m_143188_(ChunkTaskPriorityQueueSorter.java:62) ~[server-1.20.1-20230612.114412-srg.jar%23319!/:?] {re:classloading}
[03.03 15:36:13] [Server] 	at net.minecraft.util.thread.ProcessorMailbox.m_18759_(ProcessorMailbox.java:91) ~[server-1.20.1-20230612.114412-srg.jar%23319!/:?] {re:classloading}
[03.03 15:36:13] [Server] 	at net.minecraft.util.thread.ProcessorMailbox.m_18747_(ProcessorMailbox.java:146) ~[server-1.20.1-20230612.114412-srg.jar%23319!/:?] {re:classloading}
[03.03 15:36:13] [Server] 	at net.minecraft.util.thread.ProcessorMailbox.run(ProcessorMailbox.java:102) ~[server-1.20.1-20230612.114412-srg.jar%23319!/:?] {re:classloading}
[03.03 15:36:13] [Server] 	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395) ~[?:?] {}
[03.03 15:36:13] [Server] 	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?] {}
[03.03 15:36:13] [Server] 	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?] {}
[03.03 15:36:13] [Server] 	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?] {re:computing_frames}
[03.03 15:36:13] [Server] 	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?] {re:computing_frames}
[03.03 15:36:13] [Server] 	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?] {}
[03.03 15:36:13] [Server] Caused byjava.lang.ClassNotFoundException: org.apache.commons.compress.utils.Lists
[03.03 15:36:13] [Server] 	at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641) ~[?:?] {}
[03.03 15:36:13] [Server] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?] {}
[03.03 15:36:13] [Server] 	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:137) ~[securejarhandler-2.1.10.jar:?] {}
[03.03 15:36:13] [Server] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?] {}
[03.03 15:36:13] [Server] 	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:137) ~[securejarhandler-2.1.10.jar:?] {}
[03.03 15:36:13] [Server] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?] {}
[03.03 15:36:13] [Server] 	... more
```

https://github.com/jayhillx/MysticsBiomes/blob/35154f2b4961ba992114e68416c5352f377defec/src/main/java/com/mysticsbiomes/common/entity/animal/RainbowChicken.java#L70

**Comment by NikeSchuh** [comment]
Same

**Comment by jayhillx** [comment]
this is now fixed in version 3.2.
